bin/eslint.js

```diff
@@ -16,6 +16,8 @@ var cli = new CLIEngine(options);
 var debug = false;
 var checks = require("../lib/checks");
 
+console.error("ESLint is running with the " + cli.getConfigForFile(null).parser + " parser.");
```

Comment:
We already use `console.error` elsewhere in the script for our own logging, so I actually think sticking with `.error` is more consistent. The `console.log` change was made mainly (IMHO) to prevent code deep in the innards of ESLint or plugins from ruining our day. For our own code, we've mostly used `.error` so far, and personally I think we should continue to because it's more explicit. If we *were* going to switch to `.log`, though, I'd say that should be a separate refactor PR, so I'd still argue to keep this commit as-is.